framework/doc/content/source/linearfvkernels/LinearFVEnergyAdvection.md

```diff
@@ -0,0 +1,23 @@
+# LinearFVEnergyAdvection
```

Comment:
Ahhh could you move this and the corresponding documentation to the NS module? This relies on the RC interpolation which is located in the module only.

Reply:
moved

Reply:
```
ERRORS:
/LinearFVKernels/LinearFVEnergyAdvection is missing a markdown file.
  The page should be located at framework/doc/content/source/linearfvkernels/LinearFVEnergyAdvection.md
```
This is what I get

Reply:
that s because the object is still registered to MooseApp